jackmatchmaker/__init__.py

```diff
@@ -173,6 +175,10 @@ def reg_callback(self, port_id, action, *args):
         log.debug("New port registered: %s", jacklib.port_name(port))
         self._refresh()
 
+    def error_callback(self, error):
+        error = error.decode('utf-8')
+        log.info(error)
```

Comment:
Any particular reason why you use log level info here instead of error?

Reply:
Yes, because when JACK is not started, I don't want to have jack-matchmaker spam connection error in the log.

Reply:
But INFO is the default log level, so unless you use `-v WARNING|ERROR` it still will.

It's a pity that jack only passes the error message and no error code by which we could filter.

Reply:
Yes I know it's the default but that what this type of message seemed like to me...
What about making it DEBUG then?

Reply:
Agreed. Not ideal but the best easy solution.

Reply:
OK cool.